docs/getting-started.md

```diff
@@ -3,14 +3,15 @@
 To develop SpacetimeDB applications locally, you will need to run the Standalone version of the server.
 
 1. [Install](/install) the SpacetimeDB CLI (Command Line Interface)
-2. Run the start command
+2. Run the start command:
 
 ```bash
 spacetime start
 ```
 
 The server listens on port `3000` by default, customized via `--listen-addr`.
 
+💡 Standalone mode will run in the foreground: new commands will require a new terminal window.
```

Comment:
How about writing in first person: `Standalone mode will run in the foreground. Open a new terminal to run new commands`

It feels easier to parse

Reply:
That's a good one, too. In the end, I changed it to "💡 Standalone mode will run in the foreground", since there are numerous ways to do this (someone pointed out you can also add `&` to make it run in the bg). Perhaps we don't need too much info, as long as they know the gist?